### DISCUSSION TITLE
What is the "local_res" label?

### DISCUSSION THREAD

**Discussion by as0ni** [NationalSecurityAgency/ghidra#5151] [Q&A]
Hi,

Just curious what the "local_res" label refers to. I understand it represents "local reserved", but I'm looking for some more clarity. It does appear to be a value relevant beyond the local scope, and it's clearly not an argument. Example below.

![2020-01-16_16-29-36](https://user-images.githubusercontent.com/8812157/72564361-8a43fb80-387d-11ea-8228-834f1746794a.jpg)

**Comment by mumbel** [comment]
Its the difference between shadow space and local stack

**Comment by OmarDarwish** [comment]
What is shadow space?

**Comment by aa-software2112** [comment]
I'm wondering the same thing... Any idea what these "local_res" variables are? Here is another example;
![image](https://user-images.githubusercontent.com/22150653/137921643-a0023116-7582-4b7f-a32f-bbf7b951fa1b.png)

**Comment by Wall-AF** [comment]
@ryanmkurtz This hasn't been answered!

**Comment by ryanmkurtz** [comment]
Oops
